app/assets/javascripts/config_template.js

```diff
@@ -112,9 +112,9 @@ function create_editor(item) {
 function set_fullscreen(){
   $('#main').append($("#editor1"));
   $("#editor1")
-     .height($(window).height()-50)
-     .width($('#content').width())
-     .css('top', 10)
+     .height($(window).height())
+     .width($(window).width())
```

Comment:
This causes some overflow because of the padding on `#main`. you can add `.css('left', -40)` to amend this or just set to `$('#main').width()` and leave it with the padding on the sides.